app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -75,7 +75,7 @@ def import_package_profile(simple_packages)
 
       def import_package_profile_in_bulk(simple_packages)
         nvras = simple_packages.map { |sp| sp.nvra }
-        found = InstalledPackage.where(:nvra => nvras).pluck(:id, :nvra)
+        found = InstalledPackage.where(:nvra => nvras).select(:id, :nvra)
```

Comment:
what was the issue with the old code? I thought getting the non-AR items would be OK here

Reply:
the problem is down below its assuming that its an AR item, calls to .nvra